Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 // https://github.com/camunda/jenkins-global-shared-library
 // https://github.com/camunda/cambpm-jenkins-shared-library
-@Library(['camunda-ci', 'cambpm-jenkins-shared-library']) _
+@Library(['camunda-ci', 'cambpm-jenkins-shared-library@4001-oracle-23']) _
```

Comment:
Revert me when PR is reviewed and merged.

Reply:
Don't forget to revert the library branches in all Jenkins files. ☝️